pkg/apis/kops/networking.go

```diff
@@ -86,6 +86,9 @@ type WeaveNetworkingSpec struct {
 	NPCCPULimit *resource.Quantity `json:"npcCPULimit,omitempty"`
 	// NPCExtraArgs are extra arguments that are passed to weave-npc.
 	NPCExtraArgs string `json:"npcExtraArgs,omitempty"`
+
+	// Version overrides the default tag of Weave container image.
+	Version string `json:"version,omitempty"`
```

Comment:
@rifelpet any thoughts if to use Version or ImageName, seems very inconsistent?

```suggestion
	// Version overrides the default tag of Weave container image.
	Version string `json:"version,omitempty"`
```

Reply:
just for reference, i followed this logic --> https://github.com/kubernetes/kops/blob/27f9114f20f9bd49f10214c7177f474db028c4df/upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12.yaml.template#L446

Reply:
No worries, just asking @rifelpet if there's a preferred way to do this.